LeadKitAdditions/Sources/Services/CellFieldsJumpingService.swift

```diff
@@ -0,0 +1,154 @@
+import RxSwift
+import UIKit
+
+enum CellFieldsToolBarType {
+    case none
+    case `default`
+}
+
+struct CellFieldsJumpingServiceConfig {
+
+    var toolBarType: CellFieldsToolBarType = .default
+    var toolBarNeedArrows = true
```

Comment:
not let, because I want to give ability to configure it after instance creating. I don't think, that big init with all properties is a good practice.